### ISSUE TITLE
Add text labels to venue search results tabs

### ISSUE THREAD

**Issue by GoogleCodeExporter** [iamakshayv/foursquared#72]
```
After searching for a venue, on the 'Foursquare Venues' search results screen:

The 'Venues' and 'Maps' tab are missing text labels below the icons as of the 
latest release (8/30).
```

Original issue reported on code.google.com by `devin.re...@gmail.com` on 30 Aug 2009 at 4:25

**Comment by GoogleCodeExporter** [comment]
Original comment by `jlapenna` on 31 Aug 2009 at 2:37

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision c83c8d2e3e.
```

Original comment by `jlapenna` on 31 Aug 2009 at 5:19
- Changed state: **Fixed**
